charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -259,6 +259,39 @@ spec:
             secretKeyRef:
               name: {{ .Values.store.storeSecret }}
               key: "secretAccessKey"
+{{- else if eq .Values.store.storageProvider "OCS" }}
+        - name: "OCS_ENDPOINT"
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.store.storeSecret }}
+              key: "endpoint"
+        - name: "OCS_ACCESS_KEY_ID"
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.store.storeSecret }}
+              key: "accessKeyID"
+        - name: "OCS_SECRET_ACCESS_KEY"
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.store.storeSecret }}
+              key: "secretAccessKey"
+        - name: "OCS_REGION"
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.store.storeSecret }}
+              key: "region"
+        - name: "OCS_DISABLE_SSL"
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.store.storeSecret }}
+              key: "disableSSL"
+              optional: true
+        - name: "OCS_INSECURE_SKIP_VERIFY"
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.store.storeSecret }}
+              key: "insecureSkipVerify"
+              optional: true
```

Comment:
is there an option for `EnableEncryption`?

Reply:
No, this has been removed, see https://github.com/gardener/gardener-extension-provider-kubevirt/pull/59. Encryption is not supported by OCS, correct? It only made sense in the context of a "generic" S3-compatible store, but this is gone now.